WORKSPACE

```diff
@@ -29,6 +30,7 @@ load("//tensorflow:workspace.bzl", "tf_workspace")
 #    name="androidndk",
 #    path="<PATH_TO_NDK>",
 #    api_level=14)
+#Note that API level is not the same as NDK version.
```

Comment:
Sync to the latest version of what sorry? Tf? 1.1?
I will look at your comments on Monday and I'll get back to it then. I can also make similar edits to the previously mentioned files too if you want?